release-content/migration_guides.md

```diff
@@ -55,3 +55,49 @@ However, it's not always possible to use this attribute, and Bevy does not consi
 #[deprecated(since = "0.17.0", note = "This message will appear in the deprecation warning.")]
 struct MyStruct;
 ```
+
+## Style Guide
+
+Keep it short and sweet:
+
+- What, then why, then how to migrate.
+- Some helpful standardized phrases:
+  - `OldType` is now `NewType`. Replace all references and imports.
+  - The `Struct::method()` method now requires an additional `magnitude: f32` argument.
+  - `Enum` has a new variant, `Enum::NewVariant`, which must be handled during `match` statements.
+  - The `Type::method()` method has been removed. Use `Type::other_method()` instead.
+  - The `crate::old_module` module is now `crate::new_module`. Update your imports.
+  - `function()` now returns `Option<String>` instead of `String`.
+- Make sure it's searchable by directly naming the types and methods involved.
+- Use backticks for types, methods, and modules (e.g. `Vec<T>` or `core::mem::swap()`).
+- Add parentheses to the end of function names to signal they are functions (e.g. `my_function()`).
```

Comment:
I'm not personally a fan. It seems like a tradeoff between

- A user potentially confusing a function for a module
- A user potentially confusing a function with no arguments for one that takes arguments

And I think that the second case might be the one I would choose to avoid.

Reply:
Ok, I'll revert that change!